feflow/settings/integrators.py

```diff
@@ -0,0 +1,59 @@
+"""
+Settings objects for the different integrators used in the protocols.
+
+The reasoning is to have a base class that contains the settings that
+would be shared between different integrators, and subclasses of it
+for the specific integrator settings.
+"""
+
+try:
+    from pydantic.v1 import validator
+except ImportError:
+    from pydantic import validator  # type: ignore[assignment]
+
+from openff.units import unit
+from openff.models.types import FloatQuantity
+from gufe.settings import SettingsBaseModel
+
+
+class PeriodicNonequilibriumIntegratorSettings(SettingsBaseModel):
+    """Settings for the PeriodicNonequilibriumIntegrator"""
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    timestep: FloatQuantity["femtosecond"] = 4 * unit.femtoseconds
+    """Size of the simulation timestep. Default 4 fs."""
+    splitting: str = "V R H O R V"
+    """Operator splitting"""
+    equilibrium_steps: int = 250000
+    """Number of steps for the equilibrium parts of the cycle. Default 250000"""
+    nonequilibrium_steps: int = 250000
```

Comment:
Usually we put this under "SimulationSettings"

Reply:
In this case these are specific to the integrator as a single cycle/run of the integrator does a "equilibration on endstate A" -> "nonequilibrium sim on A to B (changing lambda)" -> "equilibration on endstate B"  -> "nonequilibrium sim on B to A". So these parameters are to change how many steps on each of the eq/neq parts. I hope that makes sense.

Reply:
I see what you mean. This is a bit of an odd once since, as a user, I would expect these to be more "how I define my simulation protocol" than "something related to my integrator". It probably doesn't matter, but it's something to consider longer term.